packages/vbridge/src/components/DepositConfirm/DepositConfirm.tsx

```diff
@@ -0,0 +1,228 @@
+    provider.generateNote(anchorId, destChain, amount, wrappableCurrencyAddress).then((note) => {
+      setNote(note);
+    });
+  }, [provider, anchorId, amount, destChain, activeChain, wrappableAsset]);
+  const [backupConfirmation, setBackupConfirmation] = useState(false);
+  const generatingNote = !depositPayload;
+  return (
+    <DepositInfoWrapper>
+      {generatingNote && (
+        <Loading>
+          <div>
+            <Spinner />
+          </div>
+          <Typography variant={'h4'} color={'textPrimary'} align={'center'}>
+            Generating your deposit note...
+          </Typography>
+          <Tooltip title={'close'}>
+            <CloseButton onClick={onClose} />
+          </Tooltip>
+        </Loading>
+      )}
+      <>
+        <header className={'modal-header'}>
+          <Flex row ai={'center'} jc={'center'}>
+            <CloseButton onClick={onClose} />
+          </Flex>
+
+          <Typography variant={'h4'} className={'deposit-modal-alert-header'} color={'textPrimary'}>
+            <b>Back up your note</b>
+          </Typography>
+          <SpaceBox height={8} />
+          <Typography variant={'body1'} className={'deposit-modal-alert-caption'}>
+            Please backup your note. If you lose this. <br /> you won't get your deposit back.
+          </Typography>
+          <SpaceBox height={20} />
+          <DepositAmountDecal amount={amount} symbol={provider.selectedBridgeCurrency?.view.symbol || 'UNKN'} />
+        </header>
+
+        <SpaceBox height={16} />
+
+        {loading ? null : (
+          <>
+            <GeneratedNote>
+              {note}
+              <br />
+              <Tooltip title={'Download Note'}>
+                <IconButton className={'download-button'} onClick={downloadNote}>
+                  <Icon>download</Icon>
+                </IconButton>
+              </Tooltip>
+              <Tooltip title={`Copy note the clipboard`}>
+                <CopyToClipboard onCopy={handleCopy} text={note} {...({ className: 'copy-button' } as any)}>
+                  <IconButton>
+                    <Icon>content_copy</Icon>
+                  </IconButton>
+                </CopyToClipboard>
+              </Tooltip>
+            </GeneratedNote>
+          </>
+        )}
+
+        <SpaceBox height={8} />
+
+        <FormControlLabel
+          value={backupConfirmation}
+          onChange={() => {
+            setBackupConfirmation((v) => !v);
+          }}
+          control={<Checkbox color={'primary'} style={{ color: palette.accentColor }} />}
+          label={<Typography style={{ color: palette.primaryText }}>I confirm, I backed up the note</Typography>}
+        />
+
+        <SpaceBox height={8} />
+
+        <MixerButton
+          onClick={async () => {
+            if (!depositPayload) {
+              return;
+            }
+            console.log('depositPayload: ', depositPayload);
```

Comment:
Can we remove the console log

Reply:
I actually like the console log here on the deposit payload - it shouldn't be spammy and is informative.